frontend/vite.config.js

```diff
@@ -1,6 +1,10 @@
-const path = require("path");
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
+import Cookies from "universal-cookie";
```

Comment:
Well, I know I said "Put it wherever"... But that's not the place I'd have put it in
Nvm !